apps/lib/signer/model.go

```diff
@@ -7,6 +7,7 @@ type (
 	EvmPrivateKey     string
 	StarkPublisherKey string
 	StarkPrivateKey   string
+	QuantizedPrice    string
```

Comment:
When I refactored this I was somewhat hesitant to include this because it felt kind of application specific. Is there a reason you decided to include it?